sopel/modules/currency.py

```diff
@@ -1,102 +1,189 @@
 # coding=utf-8
 # Copyright 2013 Elsie Powell, embolalia.com
+# Copyright 2019 Mikkel Jeppesen
 # Licensed under the Eiffel Forum License 2
 from __future__ import unicode_literals, absolute_import, print_function, division
 
+import time
 import re
+import requests
 
-from requests import get
-from sopel.module import commands, example, NOLIMIT
+from sopel.module import commands, example, NOLIMIT, rule
+from sopel.tools import stderr
+from sopel.config.types import StaticSection, ValidatedAttribute
```

Comment:
Should be first in this "internal" imports block. Eventually we'll start enforcing import order, and the more stuff that's already sorted correctly, the easier it will be to turn that on. :smile_cat:

Reply:
I'm not quite sure what you mean. I sorted imports according to what I think was meant, is that the correct sorting?

Reply:
Yep, this section is :ok_hand:

Reply:
Your latest changes seem to cover it. I believe the idea was that imports should not only be separated (into blocks by stdlib, internal, third-party), but also that they should be alphabetical wherever possible. So for example, `sopel.config..` should be before `sopel.module...` which is before `sopel.tools...`. I think that's what @dgw meant. ...?

Reply:
> Your latest changes seem to cover it. I believe the idea was that imports should not only be separated (into blocks by stdlib, internal, third-party), but also that they should be alphabetical wherever possible within those blocks. So for example, `sopel.config..` should be before `sopel.module...` which is before `sopel.tools...`. I think that's what @dgw meant. ...?

Oops, sorry for the late (and unnecessary) comment :grimacing: 

That's what I get for leaving the tab open all day and not refreshing... :hushed: